engine/src/main/java/com/datatorrent/stram/plan/logical/LogicalPlan.java

```diff
@@ -719,6 +719,11 @@ public void resetStreamPersistanceOnSinkRemoval(InputPortMeta sinkBeingRemoved)
         removeOperator(persistOpMeta.getOperator());
       }
     }
+
+    private void copyFrom(StreamMeta meta)
+    {
+      this.locality = meta.getLocality();
```

Comment:
This is used at only one place and it has only one statement, so why another function call?